components/IcxTopcomponent.vue

```diff
@@ -1,48 +1,51 @@
 <template>
- <div class="top-component">
-  <img src="https://drive.google.com/file/d/13m4V5FvxbQeE_u6FfBa_mtB7FijYXGOe/view?usp=sharing">
-  <div class="top-component-container">
-   <h1>
-      Globalize your company with AIESEC
-   </h1>
-   <h2>
-    　アイセックのインターンシップで、グローバルな組織作りを
-   </h2>
-  </div>
+ <div class="icx-top-container">
+  <img src="../assets/product_GT-horizontal-white.png">
+  <h1>
+    Globalize your company with AIESEC
+  </h1>
+  <h2>
+    アイセックのインターンシップで、グローバルな組織作りを
+  </h2>
```

Comment:
containerの中にもう1つdivを置いてあげると
背景の設定と見せたいコンテンツ(文字とかロゴとか)に当てたいスタイルを分けることができます！

```suggestion
<div class="icx-top-container">
    <div class="icx-top">
      <img src="../assets/product_GT-horizontal-white.png">
      <h1>
        Globalize your company with AIESEC
      </h1>
      <span>
        アイセックのインターンシップで、グローバルな組織作りを
      </span>
    </div>
  </div>
```